### PR TITLE
Add plasmamen playable species!

### PR DESCRIPTION
**Pull request by Dutch-VanDerLinde**
## About the PR
plasmamen are skeleton-like organisms that come from a planet a mostly plasma atmosphere
unlike other servers with plasmamen these guys do not combust when in contact with oxygen

they have a funny scream and the skeleton accent, similar to dwarves and lizards with their accent
accent can be removed with Accentless

they have a max age of 225 instead of 125, plasmamen age slower

their eye and skin color can be changed

naming scheme: (Element/Gas/Isotope/Metal/Other concepts) (Roman Numeral)

## Why / Balance
New skeleton-like species for more character customization

behaves exactly like every other species (no buffs/debuffs)

## Technical details
moves some folders around for better organization

## Media
image
image

https://github.com/user-attachments/assets/e7d320d8-adf5-4419-ab37-fa28a48a7d89

## Requirements

- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

:cl:
- add: Added Plasmamen as a playable species!

### CODE REVIEW

**Comment by lanse12** [comment]
cartoon sound effects ahh race

**Comment by Kadeo64** [comment]
Not complaining about another species but I feel like visually they're just slimes but opaque.

**Comment by Dutch-VanDerLinde** [comment]
> Not complaining about another species but I feel like visually they're just slimes but opaque. 

They r bones

**Comment by Tunguso4ka** [comment]
arent plasmamen is sentient plasma in the suit?

(imho, would nice to have Starbound's Novakids)

**Comment by Dutch-VanDerLinde** [comment]
> arent plasmamen is sentient plasma in the suit?
> 
> (imho, would nice to have Starbound's Novakids)

in tg station, they are skeletons that were dipped in liquid plasma being controlled by the micro-organisms inside the plasma pool

in paradise station, they are a race of skeleton-like people that live on a planet with an atmosphere high in plasma content, most of their soft tissue has fell off due to the plasma

**Comment by LankLTE** [comment]
The accent might be a bit much? I mean without the suit they’re just skeletons but multicolored, the accent kind of makes that worse while being super obnoxious (imho)

**Comment by DangerRevolution** [comment]
> Not complaining about another species but I feel like visually they're just slimes but opaque.

that's because Plasmamen were made before Slimes, but they were harder to port to 14 than Slimes so Slimes came here first

**Comment by DangerRevolution** [comment]
> The accent might be a bit much?

Accent really needs to go IMO, RMC shouldn't make some species unfun to play alongside because it's harder to read what they say

**Comment by Dutch-VanDerLinde** [comment]
> Accent really needs to go IMO, RMC shouldn't make some species unfun to play alongside because it's harder to read what they say

accent can be disabled, there are other accents that are harder to read, and the skeleton accent just adds an "ACK ACK" to the end of the sentence?? and replaces a few words

**Comment by troytroy400** [comment]
May just be me, but I feel that the emote sounds need to be made to sound less cartoony/goofy.

**Comment by Carolyn3114** [comment]
why aren't they called phoronmen?

**Comment by Dutch-VanDerLinde** [comment]
> why aren't they called phoronmen?

because it sounds stupid

**Comment by Carolyn3114** [comment]
> > why aren't they called phoronmen?
> 
> because it sounds stupid

it's even stupider to have plasmamen when plasma is a xeno thing

**Comment by Tunguso4ka** [comment]
> > why aren't they called phoronmen?
> 
> because it sounds stupid

but we dont have plasma as a gas in cm-ss13/RMC-14.
we have phoron. Phoron sheets, phoron glass, phoron gas tanks.

**Comment by QuietlyWhisper** [comment]
It's true plasma isn't used in RMC

not reviewing this yet but you can add to `Resources/Locale/en-US/_RMC14/cm-chatsan.ftl` so the species has a xenonid translation

**Review by DrSmugleaf — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/_RMC14/Species/plasmaman.yml [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+  - Unsexed
+
+- type: speciesBaseSprites
+  id: MobPlasmamanSprites
+  sprites:
+    Head: MobPlasmamanHead
+    Chest: MobPlasmamanTorso
+    Eyes: MobHumanoidEyes
+    Underwear: MobHumanoidAnyMarking
+    Undershirt: MobHumanoidAnyMarking
```
[the remark this answers is not in the document]
Comment (Kadeo64) [reply Resources/Prototypes/_RMC14/Species/plasmaman.yml:25]: i wouldn't mind it being required tbh (cause lore:tm:) but that's just me

File: Resources/Prototypes/_RMC14/Voice/speech_emote_sounds.yml [hunk 2]
```diff
@@ -222,3 +222,19 @@
...
+
+- type: emoteSounds
+  id: RMCUnisexPlasmaman
+  params:
+    variation: 0.125
+  sounds:
+    Scream:
+      collection: PlasmamanScreams
+    Honk:
+      collection: BikeHorn
```
[the remark this answers is not in the document]
Comment (Dutch-VanDerLinde) [reply Resources/Prototypes/_RMC14/Voice/speech_emote_sounds.yml:234]: This sound exists on most other humanoid species, it's so the honk reagent makes noise

[the remark this answers is not in the document]
Comment (Kadeo64) [reply Resources/Prototypes/_RMC14/Voice/speech_emote_sounds.yml:234]: no way to get honk in rmc14 without admemery and i doubt upstream will add plasmemes anytime soon

File: Resources/Prototypes/_RMC14/Datasets/Names/plasmaman_last.yml [hunk 3]
```diff
@@ -0,0 +1,102 @@
+﻿- type: dataset
```
[the remark this answers is not in the document]
Comment (Dutch-VanDerLinde) [reply Resources/Prototypes/_RMC14/Datasets/Names/plasmaman_last.yml:1]: fixed

**Comment by Dutch-VanDerLinde** [comment]
New sprites
image

**Review by DrSmugleaf — CHANGES_REQUESTED** [review]
I have to check 13 because I remember them having a sprite that looked less skeleton-y, and Whisper doesn't like them not having a chest cavity, but I might be thinking of a different codebase

**Comment by Dutch-VanDerLinde** [comment]
> I have to check 13 because I remember them having a sprite that looked less skeleton-y, and Whisper doesn't like them not having a chest cavity, but I might be thinking of a different codebase

Paradise probably

**Comment by QuietlyWhisper** [comment]
closing as stale, I don't think they fit the theme and mechanics either

**Pull request closed without merging**